nix/ouroboros-network.nix

```diff
@@ -57,14 +57,6 @@ let
           # Apply profiling arg to all library components in the build:
           enableLibraryProfiling = profiling;
 
-          # Command-line options for test suites:
-          packages.ouroboros-consensus-byron-test.components.tests.test.testFlags =
-            lib.mkForce [ "--no-create" ];
-          packages.ouroboros-consensus-shelley-test.components.tests.test.testFlags =
-            lib.mkForce [ "--no-create" ];
-          packages.ouroboros-consensus-cardano-test.components.tests.test.testFlags =
-            lib.mkForce [ "--no-create" ];
-
           packages.cardano-crypto-praos.components.library.pkgconfig =
             lib.mkForce [ [ pkgs.libsodium-vrf ] ];
           packages.cardano-crypto-class.components.library.pkgconfig =
```

Comment:
We don't need `cardano-crpyto-praos` and `cardano-crypto-class`; Network doesn't relay on any crypto stuff right now.

Reply:
Done

Reply:
Actually this is not true. `cardano-client` -> `ouroboros-consensus-diffusion` -> `ouroboros-consensus` -> crypto

Reply:
It seems nix is able to do without this.

Reply:
I meant that `nix` is able to build the shell and all packages without these lines